sdk/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: pull-migrator-image new-migration
+
+pull-migrator-image:
+	docker pull --platform linux/amd64 arigaio/atlas:latest-alpine
+
+new-migration: pull-migrator-image
+	docker run \
+		--platform linux/amd64 \
+		-v ./component/store/remote/postgresql/sqlc/migrations:/migrations \
+		--rm \
+		-it arigaio/atlas:latest-alpine \
+		migrate \
+		new $(migration_name) \
```

Comment:
we should have a check over here to make sure that the migration_name is not empty

Reply:
Fixed